cmd/bridge/main.go

```diff
@@ -129,6 +132,12 @@ func main() {
 		baseURL = bridge.ValidateFlagIsURL("base-address", *fBaseAddress)
 	}
 
+	helmChartRepoURL := &url.URL{}
+	if *fHelmChartRepoUrl != "" {
+		helmChartRepoURL = bridge.ValidateFlagIsURL("helm-chart-repo-url", *fHelmChartRepoUrl)
+		log.Infof("Helm chart repo url %v", helmChartRepoURL)
```

Comment:
I wouldn't log this since weren't not logging other URLs.

When we switch to klog, we can look at logging the various URLs at a higher log level.